platform/logging/LoggingConfig.ts

```diff
@@ -0,0 +1,166 @@
+import { Schema, typeOfSchema } from '../types';
+import { LogLevelId } from './LogLevel';
+import { Appenders, AppenderConfigType } from './appenders/Appenders';
+
+/**
+ * Separator string that used within nested context name (eg. plugins::pid).
+ */
+const CONTEXT_SEPARATOR = '::';
```

Comment:
Hm, wonder if we could use `.` as separator if we reworked the logic here https://github.com/elastic/kibana/blob/new-platform/platform/config/__tests__/ensureDeepObject.test.ts#L51-L69 to something like:

```js
test("don't expand nested properties", () => {
  const obj = {
    'foo.bar': {
      'quux.a': 1,
      b: 2
    }
  };

  expect(ensureDeepObject(obj)).toEqual({
    foo: {
      bar: {
        'quux.a': 1,
        b: 2
      }
    }
  });
});
```

Really not sure about the trade-offs here, just that `.` looks much better than `::`.